.eslintrc

```diff
@@ -7,7 +7,7 @@
         "complexity": 1,
         "no-alert": 0,
         "consistent-return": 0,
-        "max-len": 0,
+        "max-len": 0
```

Comment:
Unnecessary change. `.eslintrc` is not real json format 🍹 . 
You can have trailing commas and add comments in this file